integration_tests/integration/analytics/protectedCharacteristics.spec.ts

```diff
@@ -34,6 +35,26 @@ context('Analytics section > Protected characteristics page', () => {
       .contains('Percentage and number of prisoners in the establishment by sexual orientation')
   })
 
+  it('PC groups dropdowns are woring and they are independent', () => {
```

Comment:
worrying? 👹

Reply:
more like "working" 😬  - I'll fix the typooo

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/202/commits/fcb9f383b5482fea3796e3f59a89cc9ff1d1428d